### DISCUSSION TITLE
Option to turn off any settings so it wouldn't modify anything

### DISCUSSION THREAD

**Discussion by PinchToDebug** [valinet/ExplorerPatcher#582] [Ideas]
It would be really nice if any settings in the menus such as Taskbar, System tray etc. could be possibly turned off so they wouldn't cause bugs while it's used with other applications.

**Comment by l0oo** [comment]
What bugs you actually mean?

**Reply by PinchToDebug** [reply]
With startallback it's causing bugs, I mentioned that on wiki. Also I believe it's not the best to change anything without the user knowing it exactly. EP could be more flexible with it's "extensions".

**Reply by valinet** [reply]
EP is pretty fine. I don’t understand why it’s always EP that has to do this, EP that has to fo that. And idk what’s more flexible than source code… it also does not modify anything by default but whatever. @l0oo asked you something specific, you keep saying “bugs”. Who determined those are bugs? And why is EP to blame and not SAB?

**Reply by PinchToDebug** [reply]
@l0oo https://github.com/valinet/ExplorerPatcher/discussions/110#discussioncomment-1802904
